plugins/SegmentEditor/lang/en.json

```diff
@@ -39,6 +39,7 @@
         "SegmentOperatorIsNotNullNorEmpty": "is not null nor empty",
         "UnprocessedSegmentNoData1": "These reports have no data, because the Segment you requested %1$s has not yet been processed by the system.",
         "UnprocessedSegmentNoData2": "Data for this Segment should become available in a few hours when processing completes. (If it does not, there may be a problem.)",
+        "UnprocessedSegmentNoData3": "Matomo is unable to display data for this period, as the configuration prevents a rearchiving. Please contact your administrator or check this FAQ how to solve this problem.",
```

Comment:
Not sure if we already have a FAQ that matches for this case. If not we may need to create one and link the FAQ in the text above.